docs/UserGuide.md

```diff
@@ -205,6 +207,10 @@ Format: `editMember NAME_IN_LIST [n/NEW_NAME] [p/NEW_PHONE_NUMBER] [e/NEW_EMAIL]
 * NEW_NAME field can take on any values with alphanumeric characters.    
 * Existing values will be updated to the input values.
 
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+To prevent unintentional editing of Members, the NAME you specify must be in the currently displayed list of Members 
```

Comment:
Since we are using NAME here, we can just change NAME_IN_LIST to NAME for editMember